backend/kesaseteli/scripts/README.md

```
@@ -0,0 +1,153 @@
+
+Or from the repo root:
+
+```bash
+backend/kesaseteli/.venv/bin/python backend/kesaseteli/scripts/populate_ytj_data_to_companies.py
+```
+
+---
+
+## Typical end-to-end workflow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)